process/lifecycle.md

```diff
@@ -9,101 +9,68 @@ has_children: true
 
-* Early-stage projects that the ASWF TAC believes warrant experimentation.
-* New projects that are designed to extend one or more TAC projects with functionality or interoperability libraries.
-* Independent projects that fit the ASWF mission/vision and provide the potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
-* Projects commissioned or sanctioned by the ASWF, including initial code for ASWF Working Group collaborations, and "experimental" projects.
-* Any project that realistically intends to join ASWF Incubation in the future and wishes to lay the foundations for that.
+* Early-stage projects that the  TAC believes warrant experimentation.
+* New projects designed to extend one or more TAC projects with functionality or interoperability libraries.
+* Independent projects that fit the Academy Software Foundation mission/vision and provide the potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
+* Projects commissioned or sanctioned by the Academy Software Foundation, including initial code for Academy Software Foundation Working Group collaborations and "experimental" projects.
+* Any project that realistically intends to become an Academy Software Foundation Incubation Stage project and wishes to lay the foundations for that.
 
 ### Requirements
 
 To be accepted at the Sandbox stage, a project must:
 
-* Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC, or the TAC’s designated recipient for contribution proposals.
+* Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC or the TAC’s designated recipient for contribution proposals.
 * Provide such additional information as the TAC may reasonably request.
-* Be available to present to the TAC with respect to the project’s proposal and inclusion in ASWF. Project teams should be prepared to present a detailed (20-30 minutes in length) overview on the project in addition to speaking to the information contained in the project contribution proposal.
-* Be deemed by the TAC to add potential value or value to the mission of ASWF.
+* Present the project’s proposal and inclusion in Academy Software Foundation to the TAC. Project teams should be prepared to present a detailed (20-30 minutes in length) overview of the project and speak to the information in the contribution proposal.
+* Be deemed by the TAC to add potential value or value to the mission of Academy Software Foundation.
 * Obtain an affirmative vote of the TAC.
 
 ### Benefits
 
 The Sandbox Stage benefits are outlined below.
 
-* Neutral hosting of the project’s community and any key assets (e.g. trademark, domain, etc.).
-* Access to the ASWF collaboration infrastructure ( including GitHub, JIRA, Confluence, mailing lists, Slack ).
+* Neutral hosting of the project’s community and key assets (e.g., trademark, domain, etc.).
```

Comment:
Grammarly just told me that there's a comma after e.g. when there's more than one item in the list. You learn something every day!

Reply:
Grammarly FTW ;-)